mmv1/third_party/terraform/tests/resource_composer_environment_test.go.erb

```diff
@@ -776,12 +775,12 @@ func TestAccComposerEnvironment_withNodeConfig(t *testing.T) {
 	serviceAccount := fmt.Sprintf("tf-test-%d", RandInt(t))
 
 	VcrTest(t, resource.TestCase{
-		PreCheck:		          func() { AccTestPreCheck(t) },
+		PreCheck:		          func() { testAccPreCheck(t) },
```

Comment:
This was renamed provider-wide in https://github.com/GoogleCloudPlatform/magic-modules/pull/7639

```suggestion
		PreCheck:		          func() { AccTestPreCheck(t) },
```